### PR TITLE
Fix typos all around the codebase (generator / specification)

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

File: doc/rvv-intrinsic-spec.adoc [hunk 1]
```diff
@@ -310,13 +310,13 @@ The naming scheme does not cover all of intrinsics. Please see <<implicit-except
 Due to the limitation of the C language (without the aid of features like C++ templates), some intrinsics do not have an overloaded version. Therefore these intrinsics do not possess a simplified, EEW/EMUL-hidden interface. Please see <<unsupported-implicit-naming>> for more detail.
 
 [[implicit-exception-naming]]
-=== Exceptions in the implicit (overloaded) naming sheme
+=== Exceptions in the implicit (overloaded) naming shema
```
[the remark this answers is not in the document]
Comment (howjmay) [reply doc/rvv-intrinsic-spec.adoc]: done

File: archive/rvv-intrinsic-rfc.md [hunk 2]
```diff
@@ -1,42 +1,62 @@
 # RISC-V "V" Vector Extension Intrinsics
 
-- [Introduction](#introduction)
```
Comment (eopXD) [inline archive/rvv-intrinsic-rfc.md:3]: This is archived and users should not refer here for intrinsic specifications or behavior. Lets remove change for this file?

**Review by eopXD — COMMENTED** [review]
Except for the change in the archive, others looks good to me.

**Comment by eopXD** [comment]
Please capitalize the first letter of the commit message, and if possible add your email as contact information too in the commit just like the existing ones.

**Comment by eopXD** [comment]
Change is pushed into main with 36b4e347c1be7f91a00cedb165e6b890f7eba02b.

**Pull request closed without merging**